### PR TITLE
Export the app and version labels

### PR DESCRIPTION
**Pull request by winsock**
This ensures we are getting the app and version labels from `kube-state-metrics`. This also adds in a processor that normalizes the `app` and `version` labels.

### CODE REVIEW

File: charts/aspenmesh-collector/templates/configmap.yaml [hunk 1]
```diff
@@ -62,6 +62,19 @@ data:
...
+          - include: ^kube_.*$
+            match_type: regexp
+            action: update
+            operations:   
+              - action: update_label
+                label: label_app
+                new_label: label_app_kubernetes_io_name
+              - action: update_label
+                label: label_version
+                new_label: label_app_kubernetes_io_version
```
Comment (winsock) [inline charts/aspenmesh-collector/templates/configmap.yaml:40]: I almost like the simpler `label_app`/`label_version` label names better than the `label_app_kubernetes_io` prefixed ones. I almost wonder if we should flip the normalization, or even rename both to `app` and `version` to keep things simpler.

Comment (jizh6311) [reply charts/aspenmesh-collector/templates/configmap.yaml:40]: Will the current `metricLabelsAllowlist` be able to get all metrics with `app` and `version` labels? Can you explain more why we want to rename the labels here?

Comment (ricksalsa) [reply charts/aspenmesh-collector/templates/configmap.yaml:40]: `app` & `version` are Istio-specific. Sticking with [Kubernetes' best practices](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/) for labels will allow for other meshes to be adopted later on.

Comment (ricksalsa) [reply charts/aspenmesh-collector/templates/configmap.yaml:40]: @winsock - where are you seeing the `label_*`? It should just be `app` or `version` as an example.

Comment (winsock) [reply charts/aspenmesh-collector/templates/configmap.yaml:40]: @ricksalsa `kube-state-metrics` prefixes it with `label_` documented here(kube_deployment_labels) https://github.com/kubernetes/kube-state-metrics/blob/master/docs/deployment-metrics.md

Same applies for the pod version

Comment (ricksalsa) [reply charts/aspenmesh-collector/templates/configmap.yaml:40]: Thanks, @winsock, I found it any `kube_*_labels` in my local deployment. Moving `label_[app|version]` to `label_app_kubernetes_io_[app|version]` makes sense here.

Comment (winsock) [reply charts/aspenmesh-collector/templates/configmap.yaml:40]: Updated PR if you want to include this

**Pull request closed without merging**